office/views_admin.py

```diff
@@ -971,10 +971,21 @@ def office_edit_view(request, office_id=0, contest_office_we_vote_id=""):
     if not voter_has_authority(request, authority_required):
         return redirect_to_sign_in_page(request, authority_required)
 
+    ballotpedia_office_id = request.GET.get('ballotpedia_office_id', 0)
+    ballotpedia_office_name = request.GET.get('ballotpedia_office_name', 0)
```

Comment:
ballotpedia_office_name should have the alternate value be '' instead of 0.
The same for all of the other incoming values you are adding.